client/src/components/CategoryShow.js

```diff
@@ -45,8 +91,9 @@ const CategoryShow = (props) => {
       <div className="list-container">
         <div className="column-grid">{pastasList}</div>
       </div>
+      <div>{showForm}</div>
```

Comment:
If we're wrapping the `showForm` in a `div`, then why is it also defined with a fragment? The div can be used in place of the fragment, or remove the div here all together (especially since it's not used for styling)